libs/wingii/src/lib.rs

```diff
@@ -322,7 +332,7 @@ pub mod type_system {
 							interfaces.insert(iface);
 						}
 					}
-					interfaces.insert(iface_type);
+					interfaces.insert(iface_type.clone());
```

Comment:
Had to add a new clone here, but this is much less expensive than the ones we avoided